server/base/src/main/java/org/apache/accumulo/server/util/FileUtil.java

```diff
@@ -163,22 +163,22 @@ private static Path createTmpDir(AccumuloConfiguration acuConf, VolumeManager fs
         try {
           if (reader != null)
             reader.close();
-        } catch (IOException e) {
+        } catch (RuntimeException e) {
           log.error("{}", e.getMessage(), e);
```

Comment:
Can you update these log messages (the ones in this file that you switched from catching `IOException` to `RuntimeException`) to have a real error message (e.g. "Caught exception while closing RFile reader"), please?